public/index.html

```diff
@@ -99,6 +99,7 @@ <h2 ng-show="right.subHeading">{{right.subHeading}}</h2>
                 <div class="score lanc">LANCS {{football.lancScore}}</div>
                 <div class="score york">{{football.yorkScore}} YORK</div>
             </div>
+            	<div ng-show="football.showet" class="extratime scale-fade">ET</div>
```

Comment:
Can the attribute on football be show_extratime or something instead of showet so its more descriptive.